clcache.py

```diff
@@ -57,6 +57,16 @@
 
 HashAlgorithm = hashlib.md5
 
+
+# The codec that is used by clcache to store compiler STDOUR and STDERR in
+# output.txt and stderr.txt.
+# This codec is up to us and only used for clcache internal storage.
+# For possible values see https://docs.python.org/2/library/codecs.html
+CACHE_COMPILER_OUTPUT_STORAGE_CODEC = 'mbcs'
```

Comment:
Good question. Is "mbcs" system-dependant? I thought it is the same everywhere.

As far as I can see there is no harm in changing this to UTF-8 given that all existing content must be ASCII.